### ISSUE TITLE
MFE localization Issue

### ISSUE THREAD

**Issue by ehuthmacher** [openedx/wg-translations#16]

**Comment by ehuthmacher** [comment]
Please track progress in the following PRs:

1. https://github.com/openedx/frontend-lib-content-components/issues/385
2. https://github.com/openedx/frontend-app-communications/issues/148

**Comment by arbrandes** [comment]
The communications MFE will be ready for Quince, but not frontend-lib-content-components.

**Comment by OmarIthawi** [comment]
@arbrandes what was the problem with `frontend-lib-content-components`? It should be ready as well unless we missed anything in the process.

**Comment by brian-smith-tcril** [comment]
> It feels a bit too verbose to pull each one individually

I don't think it being verbose is necessarily a problem. It's very clear what is being pulled, and it ensures we're only pulling exactly what is needed for any given MFE.

The only part that feels a little odd is `frontend-platform` which only has 2 messages, but even that doesn't feel particularly bad.

**Comment by OmarIthawi** [comment]
> > It feels a bit too verbose to pull each one individually
> 
> I don't think it being verbose is necessarily a problem. It's very clear what is being pulled, and it ensures we're only pulling exactly what is needed for any given MFE.
> 
> The only part that feels a little odd is `frontend-platform` which only has 2 messages, but even that doesn't feel particularly bad.

We may be able to skip `frontend-platform` since their messages are duplicated in `frontend-lib-content-component`. Nevertheless, thanks for the input. I'll add it to the list of common MFE tasks.

**Comment by OmarIthawi** [comment]
I'll add those lines communications MFE to test and then we'll apply for others.

```
atlas pull \
	translations/frontend-component-header/src/i18n/messages:frontend-component-header  \
	translations/frontend-component-footer/src/i18n/messages:frontend-component-footer \
	translations/paragon/src/i18n/messages:paragon \
	translations/frontend-lib-content-components/src/i18n/messages:frontend-lib-content-components \
	translations/frontend-platform/src/i18n/messages:frontend-platform \
	translations/frontend-component-cookie-policy-banner/src/i18n/messages:frontend-component-cookie-policy-banner \
	translations/frontend-app-communications/src/i18n/messages:frontend-app-communications
```
